extension/chrome/elements/pgp_block_modules/pgp-block-attachmens-module.ts

```diff
@@ -24,9 +24,10 @@ export class PgpBlockViewAttachmentsModule {
     this.includedAtts = atts;
     for (const i of atts.keys()) {
       const name = (atts[i].name ? Xss.escape(atts[i].name) : 'noname').replace(/\.(pgp|gpg)$/, '');
+      const nameVisible = name.length > 100 ? name.slice(0, 100) + '…' : name;
       const size = Str.numberFormat(Math.ceil(atts[i].length / 1024)) + 'KB';
-      const htmlContent = `<b>${Xss.escape(name)}</b>&nbsp;&nbsp;&nbsp;${size}<span class="progress"><span class="percent"></span></span>`;
-      Xss.sanitizeAppend('#attachments', `<div class="attachment" index="${Number(i)}">${htmlContent}</div>`);
+      const htmlContent = `<b>${nameVisible}</b>&nbsp;&nbsp;&nbsp;${size}<span class="progress"><span class="percent"></span></span>`;
```

Comment:
Here you should be using `Xss.escape` when inserting untrusted content (sent by another person)

Reply:
I believe you're reviewing the outdated code.

Reply:
![image](https://user-images.githubusercontent.com/6059356/76514392-7de4b500-6460-11ea-8709-927a9321eef2.png)